django/contrib/staticfiles/finders.py

```diff
@@ -21,6 +24,17 @@ class BaseFinder(object):
     """
     A base file finder to be used for custom staticfiles finder classes.
     """
+    def check(self, **kwargs):
+        """
```

Comment:
Verify that the finder is configured correctly.

(I'm not sure if the rest is needed... maybe you could check how other `check()` methods are documented. I don't think every one of them needs to explain how the checks frameworks.)